sdk/ai/azure-ai-client/samples/endpoints/async_samples/sample_endpoints_async.py

```diff
@@ -0,0 +1,132 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
```

Comment:
This file is in the azure-ai-client folder that was deleted.

Reply:
Let me fix, it seems, I have created PR to wrong branch